parse/apk.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package parse
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/sonatype-nexus-community/nancy/types"
+)
+
+func ParseApkShow(packages []string) (projectList types.ProjectList) {
+	for _, pkg := range packages {
+		if strings.Contains(pkg, "WARNING") {
```

Comment:
Preference thing but maybe just negate this and remove the empty if block.

Reply:
In: c313d9c